scripts/check_doc_ipfs_hashes.py

```diff
@@ -21,14 +21,14 @@
 """This module contains the tools for autoupdating ipfs hashes in the documentation."""
 
 import argparse
-import json
 import re
 import sys
 from pathlib import Path
 from typing import Dict, Optional
 
 import yaml
 
+from aea.cli.packages import get_package_manager
```

Comment:
Will have to update the other projects as well https://github.com/valory-xyz/open-autonomy/issues/1526